core/routes/announcements.js

```diff
@@ -0,0 +1,93 @@
+/* @file announcements.js
+ * @brief Defines the handlers for the announcement API endpoints.
+ *
+ * @author Oscar Bezi (bezi@scottylabs)
+ */
+'use strict';
+
+var db;
+
+//==============================================================================
+// Express handlers
+//==============================================================================
+var handlers = {};
+
+//==============================================================================
+// GET/POST/DELETE /announcements
+//==============================================================================
+
+/* @brief Return every announcement. */
+handlers.get = function (req, res) {
+  var query = 'SELECT * FROM announcements';
+  db.query(query, [])
+  .then((rows) => {
+    var data = {};
+    rows = rows.map((row) => {
+      row.id = row.announcement_id;
+      delete row.announcement_id;
```

Comment:
This might not matter (I'm not sure if you plan on keeping this announcements functionality around or if it was just for testing), but what's the point of such a verbose ID field if you're just going to rename it anyway?